infrastructure/storage/src/lmdb.rs

```diff
@@ -104,6 +104,14 @@ pub struct LMDBStore {
     pub(crate) curr_db: Arc<lmdb::Database<'static>>,
 }
 
+/// Consume self to remove all databases from scope
+impl LMDBStore {
+    fn delete_db_from_scope(self) -> Result<(), DatastoreError> {
+        self.curr_db.env().sync(true)?;
+        Ok(())
```

Comment:
Can probably replace ln 110 and 111 with only ```self.curr_db.env().sync(true)```, without the question mark and semicolon.

Reply:
Question mark can be removed yes, but semicolon is needed.

Reply:
Perfect